$MirOS: ports/www/lynx/patches/patch-lynx_man,v 1.1 2008/03/09 18:19:43 tg Exp $

	site policy

--- lynx.man.orig	Mon Dec 15 00:24:33 2008
+++ lynx.man	Wed Dec 17 00:14:05 2008
@@ -207,16 +207,16 @@ it is equivalent to setting the \fBCOLOR
 library like \fBncurses\fR, Lynx relies completely on the terminal
 description to determine whether color mode is possible, and
 this flag is not needed and thus unavailable.)
-A saved show_color=always setting found in a .lynxrc file at
+A saved show_color=always setting found in a .etc/lynxrc file at
 startup has the same effect.  A saved show_color=never found
-in .lynxrc on startup is overridden by this flag.
+in .etc/lynxrc on startup is overridden by this flag.
 .TP
 .B \-connect_timeout\fR=\fIN
 Sets the connection timeout, where N is given in seconds.
 .TP
 .B \-cookie_file\fR=\fIFILENAME
 specifies a file to use to read cookies.
-If none is specified, the default value is ~/.lynx_cookies
+If none is specified, the default value is ~/.etc/cookies.txt
 for most systems, but ~/cookies for MS-DOS.
 .TP
 .B \-cookie_save_file\fR=\fIFILENAME
@@ -403,7 +403,7 @@ within a mailto URL or form ACTION.
 .TP
 .B \-nocolor
 force color mode off, overriding terminal capabilities and any
--color flags, COLORTERM variable, and saved .lynxrc settings.
+-color flags, COLORTERM variable, and saved .etc/lynxrc settings.
 .TP
 .B \-noexec
 disable local program execution. (DEFAULT)
@@ -620,7 +620,7 @@ your domain (utmp required for selectivi
 - disallow USENET News posting.
 
 .I options_save
-- disallow saving options in .lynxrc.
+- disallow saving options in .etc/lynxrc.
 
 .I outside_ftp
 - disallow ftps for people coming from outside your
@@ -780,7 +780,7 @@ Logically OR the values to combine optio
 .br
 4=TRST (table layout)
 .br
-8=config (lynx.cfg, .lynxrc, .lynx-keymaps, mime.types and mailcap contents)
+8=config (lynx.cfg, .etc/lynxrc, .lynx-keymaps, mime.types and mailcap contents)
 .br
 16=binary string copy/append, used in form data construction.
 .br
